test/track-info.js

```diff
@@ -41,5 +41,20 @@ describe('A Google Music instance playing no music', function () {
       expect(this.result).to.have.property('duration');
       expect(this.result.duration).to.be.a('number');
     });
+    describe('via API', function () {
```

Comment:
There should be a leading newline between the `it` and `describe`. It helps with the vertical rhythm of the code; delimits when separate thoughts begin.